server/src/test/java/org/cloudfoundry/identity/uaa/zone/JdbcIdentityZoneProvisioningTests.java

```diff
@@ -466,33 +469,49 @@ void testGetIdentityZoneBySubdomain() {
     }
 
     @Test
-    void testGetIdentityZoneByName() {
+    void testGetOrchestratorZoneByName() {
         IdentityZone identityZone = MultitenancyFixture.identityZone(randomValueStringGenerator.generate(), randomValueStringGenerator.generate());
         identityZone.setName("Test-Identity-Zone");
         jdbcIdentityZoneProvisioning.create(identityZone);
+        jdbcIdentityZoneProvisioning.createOrchestratorZone(identityZone.getId(), identityZone.getName());
 
-        IdentityZone retrievedIdZone = jdbcIdentityZoneProvisioning.retrieveByName(identityZone.getName());
+        OrchestratorZoneEntity orchestratorZoneEntity = jdbcIdentityZoneProvisioning.retrieveByName(identityZone.getName());
 
-        assertEquals(identityZone.getId(), retrievedIdZone.getId());
-        assertEquals(identityZone.getSubdomain(), retrievedIdZone.getSubdomain());
-        assertEquals(identityZone.getName(), retrievedIdZone.getName());
-        assertEquals(identityZone.getDescription(), retrievedIdZone.getDescription());
-        assertEquals(identityZone.getConfig().getTokenPolicy().getAccessTokenValidity(), retrievedIdZone.getConfig().getTokenPolicy().getAccessTokenValidity());
-        assertEquals(identityZone.getConfig().getTokenPolicy().getRefreshTokenValidity(), retrievedIdZone.getConfig().getTokenPolicy().getRefreshTokenValidity());
-        assertTrue(retrievedIdZone.isActive());
+        assertEquals(identityZone.getId(), orchestratorZoneEntity.getIdentityZoneId());
+        assertEquals(identityZone.getSubdomain(), orchestratorZoneEntity.getSubdomain());
+        assertEquals(identityZone.getName(), orchestratorZoneEntity.getOrchestratorZoneName());
+        assertNotNull(identityZone.getCreated());
+        assertNotNull(identityZone.getLastModified());
+        assertNotNull(orchestratorZoneEntity.getId());
```

Comment:
i guess we can add assertNotNull() checks for orchestratorZoneEntity.getCreated() and orchestratorZoneEntity.getLastModified().

Reply:
I have added assertion for orchestrator zone created and lastmodified.